uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/leave-request-popup-controller_test.js

```diff
@@ -280,7 +306,7 @@
 
         describe('when to date is selected', function () {
           beforeEach(function () {
-            $ctrl.onDateChange(new Date(), 'to');
+            setDates2016(true, true);
```

Comment:
The function definition describes the parameters and its helper function to set date only for leave
```
      /**
       * sets all dates to 2016 only
       * @param {Boolean} from set if from date is to be set
       * @param {Boolean} to set if from date is to be set
       */
```